docs/docs/development/frontend/storybook.md

```diff
@@ -0,0 +1,57 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Storybook
+
+Storybook is a tool for developing UI components in isolation. Stories are
+automatically build and pushed to Chromatic in the pipeline and visual changes
+in stories are detected automatically.
+
+In addition, Storybook can be used for easy testing of components locally.
+
+## Run storybook locally
+
+To run Storybook locally, execute the following command:
+
+```bash
+make -C frontend storybook
```

Comment:
Don't you just need to run `make storybook`?

Reply:
We don't have storybook target in the main Makefile, so `make storybook` won't work unless your current working directory is the frontend directory.